packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.base.tsx

```diff
@@ -92,23 +106,42 @@ export class ProgressIndicatorBase extends BaseComponent<IProgressIndicatorProps
       transition: percentComplete !== undefined && percentComplete < ZERO_THRESHOLD ? 'none' : undefined
     };
 
-    const ariaValueMin = percentComplete !== undefined ? 0 : undefined;
-    const ariaValueMax = percentComplete !== undefined ? 100 : undefined;
-    const ariaValueNow = percentComplete !== undefined ? Math.floor(percentComplete!) : undefined;
+    const percentCompleteAriaText = ariaValueText ? ariaValueText : this.state.percentComplete + '%';
+    const indeterminateAriaText = ariaValueText ? ariaValueText : 'Working on it';
 
     return (
       <div className={classNames.itemProgress}>
         <div className={classNames.progressTrack} />
-        <div
-          className={classNames.progressBar}
-          style={progressBarStyles}
-          role="progressbar"
-          aria-valuemin={ariaValueMin}
-          aria-valuemax={ariaValueMax}
-          aria-valuenow={ariaValueNow}
-          aria-valuetext={ariaValueText}
-        />
+        <div className={classNames.progressBar} style={progressBarStyles} />
+        {percentComplete !== undefined ? (
+          // aria-live is used in lieu of aria-valuenow/min/max and role='progressbar to
+          // circumvent the need for tabbing to hear the screen reader
+          <span aria-live="polite" className={classNames.ariaText}>
+            {percentCompleteAriaText}
+          </span>
+        ) : (
+          <span aria-live="polite" className={classNames.ariaText}>
+            {indeterminateAriaText}
+          </span>
+        )}
       </div>
     );
   };
+
+  private _throttleIntervalForAria = (nextProps: IProgressIndicatorProps): void => {
+    if (typeof nextProps.percentComplete === 'number' && typeof this.props.percentComplete === 'number') {
+      const nextPercentComplete = Math.min(100, Math.max(0, nextProps.percentComplete * 100));
+      if (
+        nextPercentComplete - this.state.percentComplete! >= 10 ||
```

Comment:
I'm a little confused with this logic.

It looks like you're clamping it between 0 and 100. And if its greater than or qual to 10%, you're updating the percent complete in the state, which will update the polite text... 

Shouldn't this do something like "only on specific intervals, like every 25%, update the state to read out a new value?)

Also, shouldn't this `Math.round`? Won't you get a number like 54.343243555 in many cases?

Reply:
This logic is happening on line 60. I can move that for you.